vnext/Desktop.ABITests/React.Windows.Desktop.ABITests.vcxproj

```diff
@@ -52,7 +52,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <SDLCheck>true</SDLCheck>
-      <AdditionalIncludeDirectories>$(VCInstallDir)UnitTest\include;$(ReactNativeDir)\ReactCommon;$(ReactNativeDir)\ReactCommon\jsi;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(VCInstallDir)UnitTest\include;$(ReactNativeDir)\ReactCommon;$(JSI_Source);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Now that RuntimeHolder.h is included here I think you need to remove RuntimeHolder.h from JSI.Shared.vcxitems and update those includes

Reply:
Done.